content/en/blog/2025/devex-survey/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Insights from the OpenTelemetry Developer Experience Survey
+linkTitle: Insights from the Developer Experience Survey
+date: 2025-03-26
+author: >
+  [Tristan Sloughter](https://github.com/tsloughter) (MyDecisiveAI), [Juliano
+  Costa](https://github.com/julianocosta89) (Datadog), [Steve
+  Gordon](https://github.com/stevejgordon) (Elastic), [Damien
+  Mathieu](https://github.com/dmathieu) (Elastic)
+sig: Developer Experience
+---
```

Comment:
You'll need to add a `cSpell:ignore:` field in the frontmatter with all the words not recognized by the SPELLING check.

Reply:
Oh ok, I see the test failure now. I thought `npm run check:text` would have told me if there were any.